components/form_inputs/file_inputs/BackgroundColorFileInput.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <v-file-input
+    background-color="blue"
+    label="File input"
+    color=black
+    autofocus
+    clearable
```

Comment:
you don't need to put this property as it's default value is "true"

Reply:
Removed